source/Network/Infrastructure/ServerConfiguration.cs

```diff
@@ -9,9 +9,9 @@ public class ServerConfiguration
         public int LanPort { get; set; } = 4201;
         public IPAddress WanAddress { get; set; } = null;
         public int WanPort { get; set; } = 4200;
-        public TimeSpan KeepAliveInterval { get; set; } = TimeSpan.FromSeconds(5);
+        public TimeSpan KeepAliveInterval { get; set; } = TimeSpan.FromSeconds(500);
         public TimeSpan LanDiscoveryInterval { get; } = TimeSpan.FromSeconds(2);
-        public TimeSpan DisconnectTimeout { get; set; } = TimeSpan.FromSeconds(15);
+        public TimeSpan DisconnectTimeout { get; set; } = TimeSpan.FromSeconds(500);
```

Comment:
I assume thats to prevent it from disconnecting while debugging? We can leave it at that for now i guess, there's not much point in kicking clients based on that.

Reply:
Client is still timing out even when this is set to 500 seconds. May create an issue for that, let me know what you think.

Reply:
Recorded in #58